static/js/components/ClassificationList.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Button } from "@material-ui/core";
+import Tooltip from "@material-ui/core/Tooltip";
+import GroupIcon from "@material-ui/icons/Group";
+import ListItem from '@material-ui/core/ListItem';
+import { FixedSizeList } from 'react-window';
+
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import * as sourceActions from "../ducks/source";
+import styles from "./ClassificationList.css";
+
+dayjs.extend(relativeTime);
+
+const ClassificationList = () => {
+  const [hoverID, setHoverID] = useState(null);
+
+  const handleMouseHover = (id, userProfile, author) => {
+    if (userProfile.roles.includes("Super admin") || userProfile.username === author) {
+      setHoverID(id);
+    }
+  };
+
+  const handleMouseLeave = () => {
+    setHoverID(null);
+  };
+
+  const dispatch = useDispatch();
+  const taxonomyList = useSelector((state) => state.taxonomies.taxonomyList);
+  const source = useSelector((state) => state.source);
+  const obj = source;
+  const userProfile = useSelector((state) => state.profile);
+  // const acls = useSelector((state) => state.profile.acls);
```

Comment:
Should this be removed?

Reply:
leaving it in as a comment because we'll need it when we add group granularity.